### ISSUE TITLE
Address bar should get focus when launching Brave

### ISSUE THREAD

**Issue by bsclifton** [brave/browser-laptop#5079]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Launching Brave does NOT set focus to the address bar unless it opens to about:newtab

**Expected behavior:**
No matter what page is loaded by default, the address bar should have focus

- Platform (Win7, 8, 10? macOS? Linux distro?):

- Brave Version:
Tested on macOS and Windows 10

- Steps to reproduce:
    1. Launch Brave
    2. If address bar doesn't have focus you're reproduced :smile: if it does have focus (and you're on about:newtab), navigate to somewhere, like brave.com
    3. Close the browser
    4. Re-launch Brave and try typing. Notice the address bar doesn't have focus

- Screenshot if needed:

- Any related issues:
https://github.com/brave/browser-laptop/issues/4464
https://github.com/brave/browser-laptop/issues/3248

**Comment by Sh1d0w** [comment]
Hm, this is not the behaviour of Chrome. I think that the UrlBar should gain focus only when Brave starts with empty new tab, to promt you to eneter address into it.

Otherwise imagine following scenario:

1. I have set Brave's home page to my fav site.
2. I launch Brave and when my home page loads instead of using down arrow to scroll my fav website I have first to use the mouse to click out of the UrlBar because the webview does not have focus because of url bar.

Same applies if you set Brave's home page to my fav search engine.

Instead of start typing directly into the loaded search engine's input in the page (like it is now), I will be forced to type in the url bar initially. Which kills the idea of having search engine as home page, no? :)

**Comment by bsclifton** [comment]
@Sh1d0w that is a great point- I didn't bother to try Chrome for comparison

cc: @bradleyrichter what do you think?

**Comment by bradleyrichter** [comment]
Yes. Sorry I was not more explicit. 

This is only for a new tab. It will happen on first time use and if the user has prefs set to open without recalling tabs from last session.

**Comment by bsclifton** [comment]
Ah ok- this should be working then :smile: I'll close this out, but please reopen if you can reproduce with 0.12.6
